src/screen/dashboard/side_menu.rs

```diff
@@ -238,9 +246,12 @@ fn buffer_button<'a>(
                 ),
             };
 
+            // Height based on font size, with some margin added.
+            let height = LineHeight::default().to_absolute((font.size + 8.0).into());
+
             button(text(content).style(theme::Text::Primary))
                 .width(length)
-                .height(length)
+                .height(height)
```

Comment:
This seems super hacky. Let me play around. We should just be able to use `Shrink` height and have it work when font size increases....